app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: 'author_id'
-  has_many :comments, foreign_key: 'author_id'
-  has_many :likes, foreign_key: 'author_id'
+  has_many :posts, foreign_key: 'user_id'
+  has_many :comments, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
```

Comment:
- [ ] You used a foreign key `user_id` while your foreign keys in the _**db/schema.rb**_ states that the foreign key is `author_id_id`. If you make the changes I suggested for other models, all you would have to do is set the foreign key to `author_id`. 👍